D4J/src/main/java/net/dv8tion/d4j/player/Bot.java

```diff
@@ -316,7 +315,7 @@ else if (message.startsWith("play "))
             {
                 String infoMsg = "";
                 String url = message.substring("play ".length());
-                Playlist playlist = Playlist.getPlaylist(url);
+                Playlist playlist = Playlist.getPlaylist(url, event.getMessage().getGuild().getID());
```

Comment:
getID is correct according to [D4J](https://github.com/nerd/Discord4J/blob/7f940d2a476a67177973eb11d9da25d01bed6c13/src/main/java/sx/blah/discord/handle/obj/Message.java#L88-L90)